Public/Src/Engine/UnitTests/Processes.Detours/FileAccessManifestTreeTest.cs

```diff
@@ -627,44 +630,32 @@ private static void TestManifestRetrieval(IEnumerable<ValidationData> validation
                         out pathId,
                         out expectedUsn);
 
-                XAssert.IsTrue(success, string.Format("Unable to find path in manifest.{0}", famDescription));
+                XAssert.IsTrue(success, "Unable to find path in manifest");
                 XAssert.AreEqual(
                     unchecked((uint)dataItem.PathId),
                     pathId,
-                    string.Format(
-                        "PathId for '{0}' did not match.{1}",
-                        dataItem.Path,
-                        famDescription));
+                    string.Format("PathId for '{0}' did not match", dataItem.Path));

```

Comment:
there is no reason calling `string.Format` here because `XAssert.AreEqual` (as well as all other `XAssert` methods) already accept `format` + `args`, so you could just write
```csharp
XAssert.AreEqual(
    unchecked((uint)dataItem.PathId),
	pathId,
	"PathId for '{0}' did not match",
    dataItem.Path);
```

Reply:
Good catch.